src/vs/workbench/contrib/testing/browser/codeCoverageDecorations.ts

```diff
@@ -60,6 +60,8 @@ const TOGGLE_INLINE_COMMAND_ID = 'testing.toggleInlineCoverage';
 const BRANCH_MISS_INDICATOR_CHARS = 4;
 
 export class CodeCoverageDecorations extends Disposable implements IEditorContribution {
+	public static readonly ID = 'editor.contrib.coverageDecorations';
```

Comment:
This is already available via the constants enum `Testing.CoverageDecorationsContributionId`